Modules/TRD/TRDQC.json

```diff
@@ -124,7 +124,7 @@
         "dataSource_comment": "no comment",
         "dataSource": {
           "type": "direct",
-          "query": "digits:TRD/DIGITS;tracklets:TRD/TRACKLETS;triggers:TRD/TRKTRGRD;tracks:TRD/MATCH_ITSTPC;trigrectrk:TRD/TRGREC_ITSTPC"
+          "query": "phdata:TRD/PULSEHEIGHT"
         },
         "taskParameters": {
           "pileupCut": "0.7",
```

Comment:
we don't need the pileup cut anymore in QC, this is a parameter in O2 now

Reply:
okay, will remove